R/DEoptim.R

```diff
@@ -1,8 +1,8 @@
 DEoptim.control <- function(VTR = -Inf, strategy = 2, bs = FALSE, NP = 50,
                             itermax = 200, CR = 0.5, F = 0.8, trace = TRUE,
                             initialpop = NULL, storepopfrom = itermax + 1,
-                            storepopfreq = 1, checkWinner = FALSE,
-                            avWinner = TRUE, p = 0.2) {

```

Comment:
Set-up around "avWinner" is removed to:

1. Catch up with DEoptim
2. As described in https://cran.r-project.org/web/packages/DEoptim/NEWS, the original set-up permits stochastic objective function where the parameters can be the same with varying optimization results. "avWinner" allows DEoptim to average the optimization results based on the same optimal parameters. This functionality could be shifted to the user to average the results over multiple calls on DEoptim via a wrapper.